src/Compiler.js

```diff
@@ -37,11 +44,11 @@ export default function Compiler(data_dir) {
       ).catch(() => {}),
     ]);
 
+    GLib.chdir(data_dir);
```

Comment:
this changes the current directory for the whole process - please use replace `GSbprocess` with  `GSubprocessLauncher` instead which supports setting cwd

https://docs.gtk.org/gio/method.SubprocessLauncher.set_cwd.html